src/Models/ProjectRepository.coffee

```diff
@@ -29,6 +29,10 @@ class ProjectRepository extends BaseModel
       @delete "projects/#{Utils.parseProjectId projectId}/repository/branches/#{encodeURI branchId}", (data) => fn data if fn
 
   # === Tags
+  addTag: (params = {}, fn = null) =>
+    @debug "Projects::addTag()"
+    @post "projects/#{Utils.parseProjectId params.id}/repository/tags", params, (data) => fn data if fn
```

Comment:
Not sure how to resolve the last line.

The first line could be resolved, but it seems that several other existing lines go over the length limit, and I wanted to stick with the existing style.